develop/parachain-devs/parachain-maintenance/runtime-upgrades.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Runtime Upgrades
+description: Explains how runtime versioning and storage migration support forkless upgrades for Polkadot SDK-based networks, and what important factors to consider when upgrading your chain.
+---
+
+Forkless runtime upgrades are a defining feature of the Polkadot SDK. The ability to update the runtime logic without forking the code base enables your blockchain to evolve and improve over time. This capability is made possible by including the definition of the runtime execution environment—the runtime WebAssembly blob—as an element in the blockchain's runtime state.
+
+
+!!!info "Why 'Forkless'?"
+    Traditional blockchains require a [hard fork](https://en.wikipedia.org/wiki/Fork_(blockchain){target=_blank}) when upgrading the state transition function of their chain.
+    A hard fork requires all node operators to stop their nodes and manually upgrade to the latest executable.
+    For distributed production networks, coordination of hard fork upgrades can be a complex process.
+
+Because the runtime is part of the blockchain state, network maintainers can leverage the blockchain's capabilities for trustless, decentralized consensus to securely make enhancements to the runtime.
+
+In the FRAME, the [`system`](https://paritytech.github.io/polkadot-sdk/master/frame_system/index.html){target=_blank} pallet defines [the `set_code` extrinsic](https://paritytech.github.io/polkadot-sdk/master/frame_system/pallet/enum.Call.html#variant.set_code){target=_blank} that is used to update the WebAssembly code for the runtime. 
+
+!!!info "Parachain vs Solo Chain Upgrades"
+    While the `set_code` extrinsic is sufficient for updating the runtime in solo chain setups, parachains must undergo a different procedure of calling `authorize_upgrade`, followed by `apply_authorized_upgrade`. These extrinsics ensure that the relay chain is aware of the upgrade, and that it gets applied
+
+When updating a runtime, there is a differentiation in _adding_ functionality to the runtime versus _updating_ existing functionality. In many cases, updating existing functionality, such as how a pallet stores state, would require a [storage migration](#storage-migration).
+
+## Runtime Versioning
+
+The component that selects the runtime execution environment to communicate with is called the _executor_.  Although you can override the default execution strategies for custom scenarios, in most cases the executor select the appropriate binary to use by evaluating the following information for both the native and WebAssembly runtime binaries:
+
+- `spec_name`
+- `spec_version`
+- `authoring_version`
```

Comment:
These three items are not introduced in the paragraph above. Maybe they need an explanation or can be deleted because they are mentioned in the table below